provisioning/ubuntu-provision.sh

```diff
@@ -672,6 +672,22 @@ function cleanup() {
   sync
 }
 
+## Install rngd to increase VM entropy
+function install_rngd() {
+  apt-get update --quiet
+  apt-get install --yes --no-install-recommends rng-tools5
+  systemctl enable rng-tools5 || true
+  systemctl rng-tools5 || true
+}
+
+## Install havegd to increase VM entropy
+function install_haveged() {
+  apt-get update --quiet
+  apt-get install --yes --no-install-recommends haveged
+  systemctl enable haveged || true
```

Comment:
```suggestion
  systemctl enable haveged || true # Uses logical operator '|| true' to prevent script from failing due to non-zero exit status in the case of docker containers
```